build.gradle

```diff
@@ -447,7 +447,7 @@ project('controller') {
         classpath += files('$APP_HOME/pluginlib')
         doLast {
             def scriptFile = file getUnixScript()
-            scriptFile.text = scriptFile.text.replace('$APP_HOME/lib/pluginlib', '$APP_HOME/pluginlib/*')
+            scriptFile.text = scriptFile.text.replace('$APP_HOME/lib/pluginlib', '$APP_HOME/pluginlib/*:$APP_HOME/pluginlib')
```

Comment:
I believe @EronWright says in the issue that this doesn't work, so why are adding rather than replacing? In other words, why do we want `$APP_HOME/pluginlib/*` here and below if it doesn't work?

Reply:
@arvindkandhare that is correct, if we want to use the classpath for providing config files to plugins, then we need the `pluginlib/` directory itself to be on the classpath.  Meanwhile, we also want the JARs to be on the classpath, hence `pluginlib/*`.